code/datums/components/itembound.dm

```diff
@@ -0,0 +1,26 @@
+/// When a movable has this component AND they are in the contents of a container, they will no longer be able to use their hands and be immobilized until they are removed from the container. So far, this is only useful for smites.
+/datum/component/itembound
+	/// The container that the subject is inside of
+	var/atom/movable/container
```

Comment:
Would it be better if this was a weakref?

Reply:
how do i weakref

Reply:
yes it would

Reply:
okay